runtime/src/main/java/org/corfudb/runtime/view/TableRegistry.java

```diff
@@ -240,6 +282,42 @@ void registerTable(@Nonnull String namespace,
         }
     }
 
+    /**
+     * For each protobuf filename -> descriptor map, validate if the previous descriptor matches
+     * If there is a new schema or an update, record the new update in the table.
+     *
+     * @param protoName - name of the protobuf file
+     * @param newProtoFd - descriptor of the protobuf file
+     * @param namespace - namespace of the table
+     * @param tableName - tablename without the namespace
+     * @return - true if schema change is detected, false otherwise
+     */
+    private boolean recordNewSchema(ProtobufFileName protoName,
```

Comment:
Why does this return a boolean if its never used?

Reply:
This was from @Lujie1996 's comments above, was trying to refactor the code, let me change the return type back to void.

Reply:
You might have not rebased since this diff doesn't look refactored and the return value still hasn't changed.

Reply:
Done, logged to one line.
It was in info level and they did not like the noise. So they asked us to move it to debug since it's only really needed for debugging and not for correctness.